src/data/reference/hi.json

```diff
@@ -460,26 +463,37 @@
+            "description": [
+                "वक्र के लिए शीर्ष निर्देशांक निर्दिष्ट करता है। इस फ़ंक्शन का उपयोग केवल <a href=\"#/p5/beginShape\">beginShape()</a> और <a href=\"#/p5/endShape\"> endShape()</a>के बीच किया जा सकता है और केवल तभी जब <a href=\"#/p5/beginShape\">beginShape()</a> के लिए निर्दिष्ट कोई मोड पैरामीटर न हो। WebGL मोड के लिए curveVertex() का उपयोग 2D में किया जा सकता है साथ ही 3D मोड। 2D मोड 2 पैरामीटर की अपेक्षा करता है, जबकि 3D मोड 3 पैरामीटर की अपेक्षा करता है। ",
+                "वक्र की शुरुआत और अंत को निर्देशित करने के लिए curveVertex() लाइनों की एक श्रृंखला में पहले और अंतिम बिंदुओं का उपयोग किया जाएगा। दूसरे और तीसरे बिंदुओं के बीच एक छोटा वक्र खींचने के लिए न्यूनतम चार बिंदुओं की आवश्यकता होती है। पांचवां जोड़ना curveVertex() के साथ बिंदु दूसरे, तीसरे और चौथे बिंदुओं के बीच वक्र खींचेगा। curveVertex() फ़ंक्शन कैटमुल-रोम स्प्लिन का कार्यान्वयन है।"
+              ],
             "params": [
                 "संख्या: x शीर्ष का समन्वय",
                 "संख्या: समन्वय और शीर्ष"
             ],
             "returns": "p5 ऑब्जेक्ट"
         },
         "endContour": {
-            "description": "'O' अक्षर के केंद्र जैसे आंकड़ों के भीतर नकारात्मक आंकड़े बनाने के लिए startContour () और endContour () फ़ंक्शन का उपयोग करें। startContour () आकृति के लिए वर्टिस की रिकॉर्डिंग शुरू करता है और एंडकंटोर () रिकॉर्डिंग समाप्त करता है। नकारात्मक आकृति को परिभाषित करने वाले कोने बाहरी आकृति के विपरीत दिशा में परिभाषित होने चाहिए। सबसे पहले घड़ी के हाथों के क्रम में बाहरी आकृति के कोने खींचें, और फिर आंतरिक आंकड़ों के लिए, लंबवत वामावर्त ड्रा करें। ये फ़ंक्शंस केवल एक शुरुआत () / एंडशेप () जोड़ी और ट्रांसफ़ॉर्मेशन जैसे ट्रांसलेशन (), रोटेट (), और स्केल () एक स्टार्टकंटोर () / एंडकंटोर () जोड़ी के भीतर काम नहीं करते हैं। अन्य आंकड़ों, जैसे कि elupse () या rect () के अंदर का उपयोग करना भी संभव नहीं है।",
+            "description": [
+                "नकारात्मक बनाने के लिए <a href=\"#/p5/beginContour\">beginContour()</a> और <a href=\"#/p5/endContour\">endContour()</a> फ़ंक्शन का उपयोग करें अक्षर 'O' के केंद्र जैसे आकार के भीतर आकार। <a href=\"#/p5/beginContour\">beginContour()</a> आकृति के लिए शीर्षों को रिकॉर्ड करना शुरू करता है और <a href=\"#/ p5/endContour\">endContour()</a> रिकॉर्डिंग बंद कर देता है। ऋणात्मक आकार को परिभाषित करने वाले शीर्षों को बाहरी आकार से विपरीत दिशा में \"wind\" होना चाहिए। पहले बाहरी दक्षिणावर्त क्रम के लिए कोने बनाएं, फिर आंतरिक के लिए आकृतियाँ, वामावर्त में शीर्ष आकृतियाँ बनाएँ।",
```

Comment:
use something better alternative for (Negative ) **नकारात्मक** !

Reply:
ऋण (minus) would be a better alternative a indicates minus value